provision-proctor/sql-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
```

Comment:
We already have a way to add the SQL secrets in the provision team which does not require a new yaml file to do a sed replace.